src/pfasLayer.js

```diff
@@ -1,156 +1,207 @@
+const PublicGoogleSheetsParser = require("public-google-sheets-parser");
```

Comment:
Could we require this instead, as it'll have been babelified and transpiled already? 

https://github.com/fureweb-com/public-google-sheets-parser/blob/main/dist/index.js

So... maybe:

```suggestion
const PublicGoogleSheetsParser = require("node_modules/public-google-sheets-parser/dist/index.js");
```

It's worth a try!

Reply:
OK, i tried this but there was an issue which I filed a fix for in the upstream library: https://github.com/fureweb-com/public-google-sheets-parser/pull/11

Reply:
If they don't merge it, we could change our package.json to point at: https://github.com/jywarren/public-google-sheets-parser/tree/patch-1